content/docs/user-guide/gems/reference/aws/aws-core/resource-mapping-tool.md

```diff
@@ -170,4 +170,4 @@ Check the [resource mapping file](/docs/user-guide/gems/reference/aws/aws-core/r
 Errors of the form ```Cannot determine region from the url``` in the logs indicate the wrong region or account has been set for the resource.
 * Double check that the resource's mapping entry has the correct region and account information, if overriding the global region and account attributes.
 * Double check that the resource's mapping entry has the correct resource type and name.
-* Ensure the mapping file sets the required global attributes for region and account. Ensure they are set to the expected values.
+* Ensure the mapping file sets the required global attributes for region and version. Ensure they are set to the expected values.
```

Comment:
Is the version decided the schema we use or do we allow developers to set it to be any valid value?

Reply:
I think the intention is that `version` reflects what schema version the file reflects w/r/t what we support. However, we [currently allow this](https://github.com/o3de/o3de/blob/development/Gems/AWSCore/Code/Include/Private/ResourceMapping/AWSResourceMappingConstants.h#L66) to be any "X.X.X" like value, so I don't think there's anything stopping developers from setting this right now to something other an what we've used in the past.

Reply:
We do not enforce version, we just enforce version format. Devs can set what they want but we use it to to show the version are tooling is generating from. Version number is owned by published json schema, which I believe is only in docs currently.